Governance/products/products-intro.md

```diff
@@ -0,0 +1,35 @@
+# Guidelines to create a climate data product
+
+## UNDER DEVELOPMENT
+
+So far, the guidelines have focused on creating, managing, and publishing climate research data with a climate science researcher as end-user. However, climate data is often used in other research fields, government initiatives and by private stakeholders for a variety of applications.
+The process to adapt and package climate data so that be of use to a wider audience, with different backgrounds and/or for different purposes is more complex than simply sharing data with other climate researchers. We know of research projects that had put together climate data products and we expect agency such as CSIRO and the Bureau of meteorology to do this on a regular basis, however there's no best practice documented available.
```

Comment:
fixed some typos:
The process of adapting and packaging climate data so that it will be of use to a wider audience, with different backgrounds and/or for different purposes is more complex than simply sharing data with other climate researchers. There are research projects that contributed to putting together climate data products, e.g., agencies such as CSIRO and the Bureau of Meteorology do this on a regular basis; However, there's no best-practice documentation available.

Reply:
I was going to make a similar correction, +1 to this one.

Reply:
done